internal/runtime/init/init.go

```diff
@@ -130,6 +130,13 @@ func LoadPaths(paths []string,
 	var result LoadPathsResult
 	var err error
 
+	var targzs, nonTargzs []string
+	if !asBundle {
+		targzs, nonTargzs = split(paths)
+		paths = targzs
+		asBundle = true
```

Comment:
This doesn't seem right, as now _all_ arguments will be treated as bundles, i.e. also directories provided to the command:

```
opa run -s bundle.tar.gz policy/
```

In this case, the bundle.tar.gz should be treated as a bundle, but not the policy directory.

Reply:
@anderseknert ,  I am checking if asBundle is false then I am making sure that .tar.gz files should be loaded as bundles. 
I ran the same command what you gave and folder given on cmd line is loaded as non-bundle. if asBundle is passed true 
(-b flag ) in the method then above code will not execute and everything will be loaded as bundle.

Reply:
Yes, if there are no .tar.gz files provided, the "paths" variable will be empty and loop (line 142) won't even execute.
I tested it with just one folder and passing -s, loads it in non-bundle mode.
basically I am doing this:

if bundle mode is false:
   simulate the bundle mode just for .tar.gz files

I added 2 test cases to my Test method (one folder , bundle mode true), (one folder, bundle mode false) and both passed. I will commit those too.

Reply:
I see, and I appreciate the added test cases. I don't think you should use loops as conditionals like that though, and again — the `asBundle` boolean will always be true in that check, making it meaningless.

You could just do something like 

```go
if asBundle || len(targzs) > 0 {
```

Right?

Reply:
Ok. That is clearer. I will change that and also add 2 test cases which I mentioned above. Appreciate the inputs.